.github/workflows/deploy_mkdocs.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy mkdocs
+on:
+  push:
+    branches:
+      - master 
+      - main
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git Credentials
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+      - uses: actions/setup-python@v4
```

Comment:
Add a step to copy to `/docs`, because:

(README.md,) LICENSE.md, SUPPORT.md and CODE_OF_CONDUCT.md must be on repo root dir, otherwise this won't work anymore:

![image](https://github.com/Ismoh/NoitaMP/assets/12631485/a9afb6bd-d46a-4a08-8414-747996b3e405)

Add a command to `deploy_mkdocs.yml` to copy those files?! I don't want to maintain copies.